index.html

```diff
@@ -0,0 +1,1005 @@
+                    <li><a href="#" class="nav-link px-2 text-secondary">Features</a></li>
+                    <li><a href="#" class="nav-link px-2 text-white">Pricing</a></li>
+                    <li><a href="#" class="nav-link px-2 text-white">Industries</a></li>
+                </ul>
+                <!--
+                <form class="col-12 col-lg-auto mb-3 mb-lg-0 me-lg-3" role="search">
+                <input type="search" class="form-control form-control-dark text-bg-dark" placeholder="Search..." aria-label="Search">
+                </form>-->
+
+                <div class="text-end">
+                    <!--<button type="button" class="btn btn-outline-light me-2">Login</button>-->
+                    <button type="button" class="btn btn-info">Request a meeting</button>
+                </div>
+            </div>
+        </div>
+    </nav>
+
+    <header class="container-fluid message">
+        <div class="row">
+            <div class="col-12 text-center">
+                <p><i><b>FlyArrow.io</b></i> is a project of the <b>ARROW DAO</b>, a non-profit decentralized organization building community owned aerial solutions. <a href="https://www.arrowair.com" target="_blank">Visit the DAO website</a></p>
+            </div>
+        </div>
+    </header>
+
+    <div class="container-fluid bg-main text-light">
+        <div class="row mb-5">
+            <div class="col-6 pe-5">
+                <h1>Building VTOL software for automated scheduled flights</h1>
+                <p>With our full-stack flightplan system you can take the pain out of launching a safe and fast system to organize and monetize on your VTOL assets and services.</p>
+                <button type="button" class="btn btn-secondary">Request a meeting</button>
+                <button type="button" class="btn btn-info">View features</button>
+            </div>
+        </div>
+        <div class="container text-center py-0">
+            <img src="/assets/img/flyarrow-software.png" alt="FlyArrow software">
+        </div>
+    </div>
+
+    <div class="container pb-5 flight-types">
+        <div class="row row-cols-1 row-cols-md-3 g-3">
+            <div class="col">
+                <div class="card p-2 rounded h-100">
+                    <div class="d-flex align-items-center">
+                        <img class="card-img" src="/assets/img/Icon-vtolcargo.svg" alt="">
+                        <div class="flex-grow-1 text-end">
+                            <span class="tag">
+                                Private + rideshare
+                            </span>
+                        </div>
+                    </div>
+                    <div class="card-body">
+                        <h3 class="card-title"> CARGO <span>flights</span></h3>
+                        <p>On demand cargo flights both private to your own users as dispatch-able flights for others. Deploy your aircraft on our network and start shipping. </p>
```

Comment:
I think this is pitching to shippers. For operators who own aircraft, we can pitch to them in the Asset Manager card. And I think these product cards should be clickable so as to direct users to the product page. 

I would reword this to:
```
Take your middle-mile logistics to the next level. Cargo deliveries are now measured by minutes. Join today to revolutionize the way you transport goods.
```